docs/installation.rst

```diff
@@ -121,3 +121,71 @@ email address. Then we could do this:
 Then you'd use the Python dotted path to that class in the
 ``settings.AUTHENTICATION_BACKENDS`` instead of
 ``mozilla_django_oidc.auth.OIDCAuthenticationBackend``.
+
+
+Creating a Django ``User`` record for new users
+-----------------------------------------------
+
+If a user logs into your site and doesn't already have an account, by default,
```

Comment:
I like that there's an "escape hatch", but I think any documentation for the setting should probably include some guidance on why you'd want to use it and how to use it.

I think I want to push off thinking about this and any resolution to another issue/PR and land this one as is in regards to `OIDC_CREATE_USER`.

Is that ok? If so, I'll write up an issue about it.